phases/1-programming.md

```diff
@@ -2,99 +2,78 @@
 
 # Phase 1 - *Source Control and Programming Basics*
 
-#### **Reading:**
-Read these books and add a 100 - 200 word review to your weekly journal. This should be a brief summary with your biggest applicable takeaway.
-
-* [Getting Real][], 37signals
-* [Passionate Programmer][], Chad Fowler
-* [Pragmatic Programmer][],  Andrew Hunt & Dave Thomas
 
 #### **Training:**
 * Week 0: CLI & Command Line
-* Week 1: Git at Sparkbox
-  * ♫ "Shunting trucks and hauling freight" ♫
-* Week 2: Functional Programming
-  * Breaking a Problem Down by Breaking a Problem Down
-  * Reason Your Way to Fame and Fortune
-* Week 3: Object Oriented Programming
-  * "Mr. Watson, come here. I want to see you." --A. Bell
-  * Cells: The Building Blocks of Life!
-* Week 4: SOLID Principles
-  * You Can't Make All the Principles Happy All of the Time
-  * Use the Force
-* Week 5: Testing
-  * "Weeks of programming can save you hours of planning"
-  * Test After: Show Correctness
-  * Test Before: Show Intention
-  * Test to Drive Design: Show Up On Time While Being Intentionally Correct
-* Week 6: Introduce CLI Project
-* Week 7: Design Patterns
-  * You Mean There's a Name for What I Did?
-* Week 8: Evolving Your Design
-  * Dammit Jim, I'm a programmer not a philosopher...oh, wait
+  * Create a bash dotfile
+  * Create at least 5 bash customizations (aliases, functions, etc.)
+  * Learn ["Command Line the Hard Way"](https://learnpythonthehardway.org/book/appendixa.html)
+  * [From Zero to Hero with Dotfiles](https://code.tutsplus.com/tutorials/setting-up-a-mac-dev-machine-from-zero-to-hero-with-dotfiles--net-35449)
+  * Read ["Wizard Zines"](https://wizardzines.com) by Julia Evans
+    * Bite Size Linux
+    * Bize Size Command Line
```

Comment:
Just a note that we were able to get free versions of these during the apprenticeship from Gary. However, each of these do cost to access.